lib/widgets/drive.coffee

```diff
@@ -51,18 +66,43 @@ getDrives = ->
 # visuals.drive('Please select a drive').then (drive) ->
 # 	console.log(drive)
 ###
+
 module.exports = (message = 'Select a drive') ->
 	getDrives().then (drives) ->
-
-		if _.isEmpty(drives)
-			throw new Error('No available drives')
-
-		form.ask
-			type: 'list'
-			name: 'drive'
+		options =
 			message: message
+			# Even though the name property doesn't have an actual use in the code,
+			# Inquirer forces us to declare it.
+			name: 'drive'
 			choices: _.map drives, (drive) ->
 				return {
 					name: "#{drive.device} (#{drive.size}) - #{drive.description}"
 					value: drive.device
 				}
+
+		ui = new UI
+			input: process.stdin
+			output: process.stdout
+		list = new InquirerList(options, ui.rl)
+		scanner = new DriveScanner getDrives,
+			interval: 4000
+			drives: drives
+
+		scanner.on 'add', (drive) ->
+			# New data about drives are automatically being added to both list.opt.choices.choices and
+			# list.opt.choices.realChoices through the use of push().
+			# https://github.com/SBoudrias/Inquirer.js/blob/master/lib/objects/choices.js#L106
+			list.opt.choices.push
+				name: "#{drive.device} (#{drive.size}) - #{drive.description}"
+				value: drive.device
+			list.render()
```

Comment:
Can you add a blank line above this? The reason is that since the comment above only applies to the `push` part, adding a blank line above this makes it clear that the comment only applies to that part.